apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/KafkaConsumerInterceptor.java

```diff
@@ -60,7 +60,7 @@ public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allA
         //
         if (records.size() > 0) {
             ConsumerEnhanceRequiredInfo requiredInfo = (ConsumerEnhanceRequiredInfo)objInst.getSkyWalkingDynamicField();
-            AbstractSpan activeSpan = ContextManager.createEntrySpan(OPERATE_NAME_PREFIX + requiredInfo.getTopics() + CONSUMER_OPERATE_NAME_SUFFIX, null).start(requiredInfo.getStartTime());
+            AbstractSpan activeSpan = ContextManager.createEntrySpan(OPERATE_NAME_PREFIX + requiredInfo.getTopics() + "/" + requiredInfo.getGroupId() +  CONSUMER_OPERATE_NAME_SUFFIX, null).start(requiredInfo.getStartTime());
```

Comment:
I have a little suggest. The OperatorName change to `Kafka/$topic/Consumer/$group.id`.

Reply:
+1

Reply:
I think the suffix "Consumer" can make a clear distinction between Consumer and Producer. How do you feel about that?

Reply:
I have carefully considered that putting topic and groupId together might cause confusion, so I will listen to your Suggestions and modify the position of groupId.